utils/misc.go

```diff
@@ -20,3 +20,11 @@ func IsNil(v interface{}) bool {
 
 	return false
 }
+
+// MinInt returns the minimum of two integers
+func MinInt(x, y int) int {
```

Comment:
only in go you write your own `min` function...

Reply:
heh